dockerbuild/run.sh

```diff
@@ -7,7 +7,7 @@ set -x
 set -e
 
 # This is a temporary fix (bug workaround) until ssp 2.0 is in use
-sed -i 's_\(\\SimpleSAML\\Error\\Assertion::installHandler()\)_// \1 _' /data/vendor/simplesamlphp/simplesamlphp/www/_include.php
+sed -i 's_\(\\SimpleSAML\\Error\\Assertion::installHandler()\)_// \1 _' /data/vendor/simplesamlphp/simplesamlphp/public/_include.php
```

Comment:
Sorry, I'll put the comment here to show what I mean. The vendor reference is changing from `www` to `public` I assume that's fine, but wanted to double-check.

Reply:
Yes, that's correct.